mltbx/brillem.prj

```diff
@@ -0,0 +1,87 @@
+<deployment-project plugin="plugin.toolbox" plugin-version="1.0">
+  <configuration file="brillem.prj" location="" name="brillem" target="target.toolbox" target-name="Package Toolbox">
+    <param.appname>brillem</param.appname>
+    <param.authnamewatermark>Gregory Tucker, Duc Le</param.authnamewatermark>
+    <param.email>duc.le@stfc.ac.uk</param.email>
+    <param.summary>Interface to the Brille library for symmetry operations and Brillouin zone interpolation.</param.summary>
+    <param.description>Interface to a C++/Python code (Brille) for symmetry operations and Brillouin zone interpolation.</param.description>
+    <param.version>TO_BE_REPLACED</param.version>
```

Comment:
Does `TO_BE_REPLACED` get automatically replaced by the package version, or is this here to be edited before the first real release?
If the latter, will the Mathworks File Exchange accept an upload with an invalid version number? If it will, is there a way to prevent accidentally publishing this there without a valid `param.version`?

Reply:
I see now that this gets automatically updated with the version number by `release.m` 🤦

Reply:
No worries...